src/lib/user/user-models.ts

```diff
@@ -21,6 +22,13 @@ export type MyLastContactedTime = {
   userLastContactedTime: Date | null;
 };
 
+export type MyAccountDetailsUpdate = {
+  userName: string;
+  userPhoneNumber: string;
+  userEmail: string;
```

Comment:
I think it will be better to exclude userEmail from the update model. It will eliminate all the questions about what should happen if email is actually different from the email in the DB. Instead, the updateMyAccountDetails action should retrieve encrypted PII from DB to get the userEmail and then do the update. It should do this within a transaction.

Reply:
Done ac89917f6805ef8e610682773b0da2f93298a9b3